src/models/all.js

```diff
@@ -0,0 +1,54 @@
+import deleteTask from './remove.js';
+
+const lists = document.getElementById('to-do-list');
+const taskArr = [];
+
+// display task
+const displayTask = (task) => {
+  const listItem = `
+  <li>
+    <div class="check">
+      <input type="checkbox" name="checkbox" class="checkbox" id="${task.description}">
+      <input type="text" class="task-description" name="${task.description}" class="task-name" id="task-name" value="${task.description}">
+    </div>
+    <div class="actions">
+      <i class="fa-solid fa-pen-to-square edit"></i>
+      <i class="fa-solid fa-trash-can del"></i>
+    </div>
+  </li>`;
+  return listItem;
+};
```

Comment:
- [ ] I am unable to delete a task because I cannot find the button. Kindly ensure the delete button is displayed and the functionality works as required. The index properties should be updated when a task is deleted.

  ![kfkfk](https://user-images.githubusercontent.com/54848363/201153805-24c442f5-2fa6-404e-b795-d1abb3906c9e.jpg)

Reply:
- [ ] As observed by the previous reviewer, there no button to delete todo on your page. Kindly look into it and add the button to delete